src/classic/clvm_tools/cmds.rs

```diff
@@ -105,22 +105,15 @@ pub fn call_tool<'a>(
             .set_help("path to clvm script, or literal script".to_string()),
     );
 
-    let mut rest_args = Vec::new();
-    for a in input_args[1..].into_iter() {
-        rest_args.push(a.to_string());
-    }
+    let rest_args: Vec<String> = input_args.iter().skip(1).cloned().collect();
```

Comment:
I dislike that simply because passing it on limits the ways in which parse_args can be used unless i also templatize the string type it uses.
tradeoff i guess.
i favor consistency on String simply to favor ownership to borrowing, but things can go either way.